1-basics/src/main.jsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react'
+import ReactDOM from 'react-dom/client'
+import {App} from './components/App'
+import './styles/global.css'
+import './styles/fonts.css'
+
+ReactDOM.createRoot(document.getElementById('root')).render(
+  <React.StrictMode>
+    <App />
+  </React.StrictMode>
+)
```

Comment:
I used to teach them with this approach so they understand variable by variable what's purpose of the file.
Also intentionally removed the StrictMode for the `basics` chapter since they already felt overwhelmed with lot of other informations

```
const div = document.getElementById('root');
const root = ReactDOM.createRoot(div);
root.render(<App />);
```

Reply:
This file needs to be JSX in Vite, otherwise it's not able to parse and render the `App` import. Agree about dropping strict mode though.